react/src/services/booksGet.js

```diff
@@ -0,0 +1,10 @@
+import api from './config';
```

Comment:
In general, we put a service per group of calls. These group are defined by the business logic, in this case, `books`. `/books` and `/books/id` could be both in the same file.
Same with `login`, `logout`, `signup`, these are referred as `auth`